lib/ui/mojo_services.cc

```diff
@@ -71,7 +64,6 @@ void MojoServices::RegisterNatives(tonic::DartLibraryNatives* natives) {
       {"MojoServices_takeIncomingServices", DartTakeIncomingServices, 0, true},
       {"MojoServices_takeOutgoingServices", DartTakeOutgoingServices, 0, true},
       {"MojoServices_takeShell", DartTakeShell, 0, true},
-      {"MojoServices_takeView", DartTakeView, 0, true},
```

Comment:
If you're going to remove this line, please also remove it from `mojo_services.dart` so we don't get a symbol resolution error at runtime if someone tries to call this function.